libraries/Microsoft.Bot.Streaming/Payloads/Assemblers/ReceiveRequestAssembler.cs

```diff
@@ -73,15 +73,15 @@ public void OnReceive(Header header, Stream stream, int contentLength)
                 stream.Position = 0;
 
                 // Execute the request on a seperate Task
-                Background.Run(() => ProcessRequest(stream));
+                Background.Run(() => ProcessRequestAsync(stream));
             }
 
             // else: still receiving data into the stream
         }
 
         public void Close() => _streamManager.CloseStream(Id);
 
-        private async Task ProcessRequest(Stream stream)
+        private async Task ProcessRequestAsync(Stream stream)
```

Comment:
>ProcessRequestAsync [](start = 27, length = 19)

cancelationToken

Reply:
The token is not available in any upstream callers and is not necessary in this private method.